lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -1054,3 +1054,58 @@ yum:
   remove:
     config_set: "install deactivate %s"
 
+vxlan_global:
+  feature:
+    config_get: "show running | i ^feature"
+    config_get_token: '/^feature fabric forwarding$/'
+    config_set: "<state> feature fabric forwarding"
+
+  dup_host_ip_addr_detection:
+    config_get: 'show running fabric forwarding all'
+    config_get_token: '/^fabric forwarding dup-host-ip-addr-detection (\d+) (\d+)$/'
+    config_set: "<state> fabric forwarding dup-host-ip-addr-detection <host_moves> <timeout>"
+
+  dup_host_ip_addr_detection_host_moves:
+    default_value: 5
+
+  dup_host_ip_addr_detection_timeout:
+    default_value: 180
+
+  anycast_gateway_mac:
+   config_get: 'show running fabric forwarding all'
+   config_get_token: '/^fabric forwarding anycast-gateway-mac (\S+)$/'
+   config_set: "%s fabric forwarding anycast-gateway-mac %s"
```

Comment:
Please use the key-value style instead of printf. We're trying to get away from the printf style.